From 3475c97ee4a6442587d86b5cd0926614c1fb2939 Mon Sep 17 00:00:00 2001
Message-Id: <3475c97ee4a6442587d86b5cd0926614c1fb2939.1607538386.git.edvin.torok@citrix.com>
From: Jonathan Davies <jonathan.davies@citrix.com>
Date: Fri, 7 Apr 2017 14:27:21 +0100
Subject: oxenstored: improve event-channel binding logging

It's useful to see a bit more detail when an inter-domain event-channel
is bound, especially over an oxenstored restart.

Signed-off-by: Jonathan Davies <jonathan.davies@citrix.com>
Reviewed-by: Wei Liu <wei.liu2@citrix.com>
Reviewed-by: Christian Lindig <christian.lindig@citrix.com>
Release-acked-by: Julien Grall <julien.grall@arm.com>

diff --git a/tools/ocaml/xenstored/domain.ml b/tools/ocaml/xenstored/domain.ml
index 41db7e44d4..081076271a 100644
--- a/tools/ocaml/xenstored/domain.ml
+++ b/tools/ocaml/xenstored/domain.ml
@@ -71,7 +71,7 @@ let notify dom = match dom.port with
 
 let bind_interdomain dom =
 	dom.port <- Some (Event.bind_interdomain dom.eventchn dom.id dom.remote_port);
-	debug "domain %d bound port %s" dom.id (string_of_port dom.port)
+	debug "bound domain %d remote port %d to local port %s" dom.id dom.remote_port (string_of_port dom.port)
 
 
 let close dom =
